src/services/authentication/strategies/JwtStrategy.js

```diff
@@ -0,0 +1,12 @@
+const { JWTStrategy } = require('@feathersjs/authentication');
+const { ObjectId } = require('mongoose').Types;
+
+class CustomJwtStrategy extends JWTStrategy {
```

Comment:
This is the default, or for the creating of the support JWTs?

Reply:
This is the default. I had to overwrite `getEntity` to keep the strategy from using the legacy account service.